src/main/java/uk/gov/digital/ho/hocs/document/model/DocumentData.java

```diff
@@ -96,6 +96,16 @@ public void update(String fileLink, String pdfLink, DocumentStatus status) {
         this.updated = LocalDateTime.now();
     }
 
+    public void update(DocumentStatus status) {
+        if (status == null) {
+            throw new ApplicationExceptions.EntityCreationException("Cannot call DocumentData.update",
```

Comment:
More of a nitpick but is `EntityCreationException` the right exception? Might be worth also saying that the status is null for when we're looking in logs?

Reply:
Yes, I'll update this and the original update method